dev/storage.h

```diff
@@ -2843,7 +2737,87 @@ namespace sqlite_orm {
             }
             
         protected:
+            template<class O>
+            unsigned int fill_statement_for_each_column(const O& o, unsigned int index, sqlite3_stmt* const stmt) {
+                auto& impl = get_impl<O>();
+                impl.table.for_each_column([&o, &index, &stmt, &impl] (auto c) {
+                    using field_type = typename decltype(c)::field_type;
+                    if(c.member_pointer) {
+                        statement_binder<field_type>().bind(stmt, index++, (o.*c.member_pointer));
+                    } else {
+                        statement_binder<field_type>().bind(stmt, index++, ((o).*(c.getter))());
+                    }
+                });
+
+                return index;
+            }
+
+            template<class O>
+            unsigned int fill_statement_for_each_non_primary_key_column(const O& o, unsigned int index, sqlite3_stmt* const stmt) {
+                auto& impl = get_impl<O>();
+                impl.table.for_each_column([&o, &index, &stmt, &impl] (auto c) {
+                    using field_type = typename decltype(c)::field_type;
+					if (!c.template has<constraints::primary_key_t<>>()) {
+                         if(c.member_pointer) {
+                             statement_binder<field_type>().bind(stmt, index++, (o.*c.member_pointer));
```

Comment:
does it work with composite key?

Reply:
@fnc12 it is like before, just refactor

Reply:
@fnc12 Did you want to remove no_primary_key functions?

Reply:
@soroshsabz Ok, but I do not understand you last comment